odl/discr/discr_ops.py

```diff
@@ -151,6 +159,226 @@ def adjoint(self):
         """
         return self.inverse
 
+
+class ResizingOperator(Operator):
+
+    """Operator mapping between discretized spaces of different shapes.
+
+    This operator is intended as a mapping between uniformly
```

Comment:
"is intended as" is abit unnecessary, unless this is an abstract operator intended for inheritance.